bindings/ocaml/lib/operator.ml

```diff
@@ -18,12 +18,30 @@
 *)
 
 let new_operator = Opendal_core.Operator.operator
+let stat = Opendal_core.Operator.stat
 let is_exist = Opendal_core.Operator.blocking_is_exist
 let create_dir = Opendal_core.Operator.blocking_create_dir
 let read = Opendal_core.Operator.blocking_read
+let reader = Opendal_core.Operator.blocking_reader
```

Comment:
Since it could be a very long time before we add async support for ocaml binding. How about removing all `blocking_` prefix?

Reply:
good idea

Reply:
If all the blocking prefixes are removed, some method names may be `write` and `read`, which will cause an abnormal error in FFI, maybe we should still rename them in the generated code

Reply:
> Wouldn't it look weird if we kept the `blocking_` prefix only for these two special functions?

Yes, it looks weird.

Reply:
Maybe we don't need to change the `blocking_` prefix, because this only exists in the internal module of `Opendal_core`, and users will only use the methods in the `Operator` module, these methods do not have the `blocking_` prefix.